client/my-sites/patterns/components/pattern-gallery/client.tsx

```diff
@@ -113,15 +114,20 @@ const PATTERNS_PER_PAGE_COUNT = 9;
 
 export const PatternGalleryClient: PatternGalleryFC = ( props ) => {
 	const {
-		category,
 		displayPlaceholder = false,
 		getPatternPermalink,
 		isGridView = false,
```

Comment:
While it is technically carried by `PatternsContext`, I've left `isGridView` as a prop here, because it will still need to be accessible to `pattern-gallery/server.tsx`, which isn't using `PatternsContext`